Source/WebKit/PlatformWPE.cmake

```
@@ -296,6 +296,20 @@ list(APPEND WebKit_LIBRARIES
 )
 
 WEBKIT_BUILD_INSPECTOR_GRESOURCES(${DERIVED_SOURCES_WEBINSPECTORUI_DIR})
+
+if (ENABLE_BREAKPAD)
+    list(APPEND WebKit_INCLUDE_DIRECTORIES
+        ${BREAKPAD_INCLUDE_DIRS}
+        ${WEBKIT_DIR}/Shared/unix
+    )
+    list(APPEND WebKit_LIBRARIES
+        ${BREAKPAD_LIBRARIES}
+    )
```

Instead of setting both variables, `Breakpad` could become a target created in its `Find` module. (More details there)

```suggestion
    list(APPEND WebKit_LIBRARIES
        Breakpad::BreakPad
    )
```